client/my-sites/domains/domain-management/name-servers/style.scss

```diff
@@ -6,14 +6,17 @@
 
 .name-servers__dns {
 	.name-servers__toggle {
-		display: inline-block;
-		float: right;
-
-		.components-toggle-control .components-base-control__field .components-form-toggle {
-			margin-right: 0;
+		.components-base-control__field {
+			justify-content: space-between;
+			margin-bottom: 0;
```

Comment:
I'd suggest removing the `margin-bottom: 0`. It adds a bit more spacing at the bottom than what we used to have, but to me, it looks fine and doesn't look so packed. So it's a good compromise IMHO.